frontend/src/embedding-sdk-bundle/components/public/dashboard/EditableDashboard/enterprise.unit.spec.tsx

```diff
@@ -10,26 +10,29 @@ import { screen, waitFor, within } from "__support__/ui";
 import { ROOT_COLLECTION } from "metabase/entities/collections";
 import { createMockCollection } from "metabase-types/api/mocks";
 
-import { addSubscriptionTests } from "../shared-tests/subscriptions.spec";
+import { addEnterpriseSubscriptionsTests } from "../shared-tests/subscriptions.spec";
 import {
   type SetupSdkDashboardOptions,
   setupSdkDashboard,
 } from "../tests/setup";
 
 import { EditableDashboard } from "./EditableDashboard";
 
-const setup = async (
+const setupEnterprise = async (
   options: Omit<SetupSdkDashboardOptions, "component"> = {},
 ) => {
   return setupSdkDashboard({
     ...options,
+    enterprisePlugins: ["sdk_subscriptions", "embedding"],
```

Comment:
Two thing to look for all these `enterprise.unit.spec.*` and `premium.unit.spec.*` are:
1. `enterprisePlugins` are always provided. They might not need to same feature set
2. Only `premium.unit.spec.*` will contain `tokenFeatures`. This will behave like when users have (a) valid token feature(s)